biz.aQute.bndlib/src/aQute/bnd/osgi/PluginsContainer.java

```diff
@@ -145,10 +212,6 @@ public boolean remove(Object plugin) {
 		return plugins().spliterator();
 	}
 
-	@Override
-	public Stream<Object> stream() {
```

Comment:
PluginContainer is a Set<Object> so it must have a valid `stream()` implementation. You removed this which means it falls back to the implementation in Collection which in turn depends on the spliterator method which does not handle the changes here. So the set contact is still wrong. You can't view all the plugins in the set that can be see by the `stream(Class)` method. All the set operations need to see all the plugins in the set.